README.md

```diff
@@ -0,0 +1,58 @@
+# seniormap
+An interactive map of GMHS seniors' college/postsecondary destinations.  Created by Erik Boesen (GMHS Class of 2019), the map is now maintained by the GMHS Advanced Programming Club. If you'd like to make a submission, please visit [the map](https://apc-gm.com/seniormap) and click on the form in the bottom right corner.
+
+## Maintainence instructions 
+### Creating a the form and spreadsheet for a new year
+
+1. Navigate to the ['Senior Map' folder](https://drive.google.com/drive/folders/1Jko-Gei3H9em6nXjL_Tia8j4NNptmD6k) on Google Drive and create a new folder with the year of the graduating class (e.g. 2027). Request access to the folder if you do not already have access.
+2. Make a copy of a previous year's form and edit content accordingly for the appropriate year.
+3. Ensure the form has a linked Google spreadsheet with the appropriate fields.
+![Screenshot of creating linked spreadsheet](resources/readme/tut1.png)
+4. Publish the spreadsheet to the web (File > Publish to the web; link will then be provided).
+
+### Adding the new year to the code base
+
+1. Within `js/scripts.js`, search for "README FLAG" and follow the instructions to add an entry, with the spreadsheet's datasheet URL for the appropriate year.
+   - For example: `['2027', 'a8a8a8a8a8a8a8a8a8a8a8a8a8a'],`
+2. Create an entry for the student submission spreadsheet you created previously in the 'Year Data' sheet (Senior Map/Shared Data/Year Data). Mimick the existing entry cells and paste the datasheet URL next to the year you created. 
+3. In the same spreadsheet, create an entry for the corresponding Google form submission link in the right adjacent cell. This link appear on the map so students can submit their post-secondary destination. 
+![Screenshot of 'Year Data' cells](resources/readme/tut5.png).
+
+### Obtaining and readying senior portraits of students
+
+1. Email the GMHS registrar, Lynette Kemp, for a list of students in your grade. The registrar is the administrator who manages student enrollment and will have this information on hand; their contact information is listed in the [GMHS staff directory](https://www.fccps.org/o/gmhs/staff?filter_id=%5B68455%5D).
+2. Email the yearbook club teacher lead and ask for senior portraits for the senior map. They will share a folder with you on Google Drive containing all the portraits.
+   - These images will be in alphabetical order with file names of 00001.jpeg, 00002.jpeg, etc.
+   ![Screenshot of numbered senior portraits](resources/readme/tut6.png)
+3. Download all of the photos from Google Drive into a folder on your desktop.
+4. Download the photo naming script available at [`resources/tools/namePictures.py`](https://github.com/apc-gmhs/seniormap/blob/gh-pages/resources/tools/namePictures.py).
```

Comment:
Replace link with just `resources/tools/namePictures.py`. You don't need the whole link when you're reading on GitHub.